content/php/concepts/math-functions/terms/tanh/tanh.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'tanh()'
+Description: 'Returns the hyperbolic tangent of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`tanh()`** function in PHP is a mathematical function that returns the hyperbolic tangent of a number. It calculates the ratio of the hyperbolic sine to the hyperbolic cosine of the input value.
+
+## Syntax
+
+```pseudo
+$result = tanh($number);
+```
+
+`$number` - the input value for which the hyperbolic tangent will be returned.
+
+## Example
+
+In the following example the `tanh()` function is used to print the tanh of `10`:
+
+```php
+<?php
+
+  echo(tanh(10));
+
+?>
+```
+
+The output will be:
+
+```shell
+0.99999999587769
+```
+
+## Codebyte Example
```

Comment:
I think this second example isn't really demonstrating anything new about how this function works. Please remove it unless it can be updated to provide some additional value about the use of tanh.